nemo/utils/callbacks/dist_ckpt_io.py

```diff
@@ -270,7 +270,10 @@ def load_checkpoint(
             sharded_state_dict = self.adjust_non_strict_load(path, sharded_state_dict)
 
         return dist_checkpointing.load(
-            sharded_state_dict=sharded_state_dict, checkpoint_dir=path, sharded_strategy=sharded_strategy
+            sharded_state_dict=sharded_state_dict,
+            checkpoint_dir=path,
+            sharded_strategy=sharded_strategy,
+            validate_access_integrity=False,
```

Comment:
@paul-gibbons We can't turn it off always, only when absolutely necessary because of irregular load.
Can you parametrize it?